src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -25,5 +25,21 @@
 
 @RunWith(JUnit4.class)
 public final class UtilityTest {
-  
+  @Test
+  public void convertUsingGsonToJsonTest() {
+    // Object with attributes.
+    SubjectTag tag = new SubjectTag(5, "Interviews", "red");
+    String actual = Utility.convertToJsonUsingGson(tag);
+    String expected = "{\"id\":5,\"subject\":\"Interviews\",\"color\":\"red\"}";
+    Assert.assertEquals(expected, actual);
+  }
+
+  @Test
+  public void convertUsingGsonToJsonEmptyTest() {
+    // Empty object.
```

Comment:
Can you note that the JSON will therefore be populated with default values?

Reply:
Sure!